src/main/scala/it/pagopa/interop/tenantprocess/api/impl/TenantApiServiceImpl.scala

```diff
@@ -93,7 +94,7 @@ final case class TenantApiServiceImpl(
 
     val result: Future[Tenants] = for {
       requesterId <- getOrganizationIdFutureUUID(contexts)
-      result      <- TenantReadModelQueries.listConsumers(name, requesterId, offset, limit)(readModel)
+      result      <- ReadModelTenantQueries.listConsumers(name, requesterId, offset, limit)
```

Comment:
shouldn't this go to TenantManagementService?

Reply:
No this went directly to ReadModel, but if you want I will pass for mgmt?